docs/usage/shoot_serviceaccounts.md

```diff
@@ -67,7 +67,7 @@ If uncertain that these prerequisites are met, please contact your Gardener Admi
 
 ### Enablement
 
-If the prerequisites are met then the feature can be enabled for a shoot cluster by annotating it with `authentication.gardener.cloud/issuer=managed`. Mind that once enabled, this feature cannot be disabled. After the shoot is reconciled, you can retrieve the new shoot service account issuer value from the shoot's status. A sample query that will retrieve the managed issuer looks like this:
+If the prerequisites are met then the feature can be enabled for a shoot cluster by annotating it with `authentication.gardener.cloud/issuer: managed`. Mind that once enabled, this feature cannot be disabled. After the shoot is reconciled, you can retrieve the new shoot service account issuer value from the shoot's status. A sample query that will retrieve the managed issuer looks like this:
```

Comment:
Although I understand that a direct edit to the shoot's yaml requires the key/value of annotation to be split by ":", the kubectl command on the other hand uses `key=value` format. Having an "=" sign there is also consistent with other gardener documentation so I would not change it here.
```suggestion
If the prerequisites are met then the feature can be enabled for a shoot cluster by annotating it with `authentication.gardener.cloud/issuer=managed`. Mind that once enabled, this feature cannot be disabled. After the shoot is reconciled, you can retrieve the new shoot service account issuer value from the shoot's status. A sample query that will retrieve the managed issuer looks like this:
```